tests/components/brottsplatskartan/test_config_flow.py

```diff
@@ -0,0 +1,224 @@
+"""Test the Brottsplatskartan config flow."""
+from __future__ import annotations
+
+from unittest.mock import patch
+
+from homeassistant import config_entries
+from homeassistant.components.brottsplatskartan.const import CONF_AREA, DOMAIN
+from homeassistant.const import CONF_LATITUDE, CONF_LOCATION, CONF_LONGITUDE
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import (
+    RESULT_TYPE_ABORT,
+    RESULT_TYPE_CREATE_ENTRY,
+    RESULT_TYPE_FORM,
+)
+
+from tests.common import MockConfigEntry
+
+
+async def test_form(hass: HomeAssistant) -> None:
+    """Test we get the form."""
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == RESULT_TYPE_FORM
+    assert result["errors"] == {}
+
+    with patch(
+        "homeassistant.components.brottsplatskartan.config_flow.uuid.getnode",
+        return_value="1234567890",
+    ), patch(
+        "homeassistant.components.brottsplatskartan.async_setup_entry",
+        return_value=True,
+    ) as mock_setup_entry:
```

Comment:
Could `mock_setup_entry` be moved to a fixture in `conftest.py`, and auto-used inside `tests_config_flow.py` with `pytestmark = pytest.mark.usefixtures("mock_setup_entry")`?